components/script/timers.rs

```diff
@@ -209,11 +209,11 @@ impl OneshotTimers {
         loop {
             let mut timers = self.timers.borrow_mut();
 
-            if timers.is_empty() || timers.last().unwrap().scheduled_for > base_time {
+            if timers.is_empty() || timers.back().unwrap().scheduled_for > base_time {
                 break;
             }
 
-            timers_to_run.push(timers.pop().unwrap());
+            timers_to_run.push(timers.pop_front().unwrap());
```

Comment:
`pop_front` removes the last element of the `VecDeque`, but `pop` removes the first element of the `Vec`. Since the code doesn't need to remove the first element ever, maybe the benefit of `VecDeque` is minimal.

Reply:
Sorry, I had reversed what I meant to say. `VecDeque::pop_front()` removes the first element and `Vec::pop` removes the last element, which results in different behavior here.

Reply:
How does performance look with this change?